Gopkg.toml

```diff
@@ -30,12 +30,8 @@
   name = "github.com/bluele/slack"
 
 [[constraint]]
-  name = "github.com/sensu/sensu-go"
-  revision = "2daf9d442deec0afd2c6f53c183460e879a10646"
-
-[[constraint]]
-  name = "github.com/spf13/cobra"
-  version = "0.0.3"
+  name = "github.com/sensu/sensu-plugins-go-library"
+  version = "0.2.0-alpha1"
```

Comment:
As of 15 minutes this can be updated to `0.2.0-alpha4`

Reply:
I take that back you can now use `0.2.0` since the changes have now been merged to master.